Syntax error in action.yml

There is a small typo in packages/automatic-releases/action.yml.

##[error]marvinpinto/action-automatic-releases/latest/action.yml (Line: 27, Col: 5): Unexpected value 'decription'
See: https://github.com/libretro/mame/runs/942138167?check_suite_focus=true

Strange is that I got no error in my repo: https://github.com/tcamargo/mame/runs/941527950?check_suite_focus=true

Anyway, should be an easy fix.

--- a/packages/automatic-releases/action.yml
+++ b/packages/automatic-releases/action.yml
@@ -24,7 +24,7 @@ inputs:
     required: false
 outputs:
   automatic_releases_tag:
-    decription: "The release tag this action just processed"
+    description: "The release tag this action just processed"
 runs:
   using: "node12"
   main: "dist/index.js"

